### DISCUSSION TITLE
Does Vercel support HTTP/2 Next.js hosting?

### DISCUSSION THREAD

**Discussion by nbouvrette** [vercel/vercel#6764] [General] [answered]
I found this old discussion around HTTP2 https://github.com/vercel/next.js/discussions/10842 which I am not sure is still relevant.

When I run a Lighthouse report on one of my projects, it suggests using HTTP2 to speed up performance and I could not find any way to configure this on Vercel.

I also found this https://github.com/vercel/next.js/issues/12159 which seems to contradict the first discussion. If this one is actually more up-to-date, is there any plan to prioritize this as this can have a considerable impact on performance?

**Comment by leerob** [comment, marked as answer]
Yes, [this comment](https://github.com/vercel/next.js/discussions/10842#discussioncomment-840) is still accurate. Vercel supports HTTP/2 and will use it whenever possible. We currently do not support [Push](https://en.wikipedia.org/wiki/HTTP/2_Server_Push) yet, however.

https://vercel.com/docs/concepts/edge-network/overview#supported-protocols

**Reply by leerob** [reply]
For example, here is my site https://leerob.io.

![CleanShot 2021-09-23 at 11 44 17@2x](https://user-images.githubusercontent.com/9113740/134549284-0ef2da30-42aa-4f76-979c-908f35e25cc8.png)
